pom.xml

```
@@ -21,7 +21,7 @@
   <properties>
     <revision>1.46</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.222</jenkins.version>
+    <jenkins.version>2.249</jenkins.version>
```

use the actual LTS version?

```suggestion
    <jenkins.version>2.249.2</jenkins.version>
```

---

Not sure JCasC is ready to bump to such a recent LTS?

---

I would not mind, but I am ready to pause this patch if there is a concern. There were no security releases in 2.249.x , so one can safely stay on 2.235

---

ah I thought this was integrations, yeah 2.235 might be better

---

2.249 should be fine now, but can we use the actual LTS?